apollo-router/src/plugins/traffic_shaping/mod.rs

```diff
@@ -67,7 +68,7 @@ struct Shaping {
     #[schemars(with = "String", default)]
     /// Enable timeout for incoming requests
     timeout: Option<Duration>,
-    //  *experimental feature*: Enables request retry
+    /// Retry configuration
```

Comment:
this should still be marked as experimental for now, even in the documentation

Reply:
[be692aa](https://github.com/apollographql/router/pull/2173/commits/be692aa5df7c988bcd97acceff6153b801bfacc5)